getting_started/intel_debug/introduction.rst

```diff
@@ -117,13 +117,64 @@ Linux kernel to query whether or not the firmware authentication is enabled,
 which means `dmesg` logs cannot be provided to alert the user to an ME
 configuration issue.
 
+Linux SOF will look up firmware files at following paths:
+
+.. list-table:: Firmware look-up paths per Intel platform
+   :widths: 50 50 25
+   :header-rows: 1
+
+   * - Platform
+     - Firmware load path
+     - Notes
+   * - Raptor Lake and older
+     - /lib/firmware/intel/sof/sof-PLAT.ri
+     - PLAT = glk, cml, ..., rpl
+   * - Raptor Lake and older (community signed)
+     - /lib/firmware/intel/sof/community/sof-PLAT.ri
+     - PLAT = glk, cml, ..., rpl
+   * - Meteor Lake and newer
+     - /lib/firmware/intel/sof-ipc4/PLAT/sof-PLAT.ri
+     - PLAT = mtl, lnl, ...
+   * - Meteor Lake and newer (community signed)
+     - /lib/firmware/intel/sof-ipc4/PLAT/community/sof-PLAT.ri
+     - PLAT = mtl, lnl, ...
+   * - Meteor Lake and newer Loadable Module
+     - /lib/firmware/intel/sof-ipc4-lib/PLAT/UUID.bin
+     - PLAT as above, UUID = UUID of the module
+   * - Meteor Lake and newer Loadable Module (community signed)
+     - /lib/firmware/intel/sof-ipc4-lib/PLAT/community/UUID.bin
+     - PLAT as above, UUID = UUID of the module
+
+Important notices:
+ - Standard Linux firmware search path and order is followed. The above table covers the base "/lib/firmware" case. Please see https://docs.kernel.org/driver-api/firmware/fw_search_path.html for more information.
+ - The firmware folder and filename can be overridden with "fw_path" and "fw_filename" SOF kernel parameters.
+ - The loadable module library path can be overridden with "lib_path" SOF kernel parameter.
+
 2. Topology file
 ----------------
 
 The topology file, such as ``/lib/firmware/intel/sof-tplg/sof-hda-generic-2ch.tplg``, describes the processing graph and controls to
 be instantiated by the SOF driver. The topology can be regenerated and
 reconfigured with tools but requires expert knowledge of the ALSA/ASoC/topology frameworks.
 
+.. list-table:: Firmware topology file look-up paths per Intel platform
+   :widths: 50 50 25
+   :header-rows: 1
+
+   * - Platform
+     - Topology load path
+     - Notes
+   * - Raptor Lake and older
+     - /lib/firmware/intel/sof-tplg/sof-MACH.tplg
+     - MACH = machine driver
+   * - Meteor Lake and newer
+     - /lib/firmware/intel/sof-ace-ipc/sof-MACH.tplg
+     - MACH = machine driver
```

Comment:
not really, this is an indication but not the name of the machine driver, we have e.g. multiple permutations of the hardware supported by the same machine driver but different topology files.

Reply:
@plbossart true, but couldn't come up with better. maybe I'll put something very generic like CONFIG.

Reply:
modified to CONFIG